sdcm/cluster.py

```diff
@@ -2211,13 +2211,13 @@ def _gen_cqlsh_cmd(self, command, keyspace, timeout, host, port, connect_timeout
             auth_params=auth_params, use_keyspace=use_keyspace, timeout=timeout, connect_timeout=connect_timeout, ssl_params=ssl_params)
         return 'cqlsh {options} -e "{command}" {host} {port}'.format(options=options, command=command, host=host, port=port)
 
-    def run_cqlsh(self, cmd, keyspace=None, port=None, timeout=60, verbose=True, split=False, target_db_node=None, connect_timeout=5):
+    def run_cqlsh(self, cmd, keyspace=None, port=None, timeout=120, verbose=True, split=False, target_db_node=None, connect_timeout=5):
```

Comment:
Do we need to change it for all tests/functions?

Reply:
I don't see a reason not to...